src/components/EditBlueprint.js

```diff
@@ -360,12 +366,15 @@ class EditBlueprint extends PureComponent
 	actuallySaveBlueprintEdits = async () =>
 	{
 		const imageUrl = this.state.blueprint.imageUrl;
-
-		const goodRegex1 = /^https:\/\/imgur\.com\/([a-zA-Z0-9]{7})$/;
-		const match      = imageUrl.match(goodRegex1);
-		if (!match)
+		
+		const regexPatterns = {
+			imgurUrl1: /^https:\/\/imgur\.com\/([a-zA-Z0-9]{7})$/,
+			imgurUrl2: /^https:\/\/i\.imgur\.com\/([a-zA-Z0-9]+)\.[a-zA-Z0-9]{3,4}$/,
+		};
+		const matches       = Object.values(regexPatterns).map(pattern => regexCheck(pattern)).filter(Boolean);
+		if (matches.length <= 0)
 		{
-			console.log('Create.actuallyCreateBlueprint error in imageUrl', {imageUrl});
+			console.log('EditBlueprint.actuallySaveBlueprintEdits error in imageUrl', {imageUrl});
 			return;
 		}
 
```

Comment:
```suggestion

const match = matches[0];
```

unless I'm missing something, `match` is undefined here.

Reply:
I added that line and pushed to the branch since you gave me access.